### PR TITLE
create hbr731.md

### PR DESCRIPTION
**Pull request by hbr731**
https://raw.githubusercontent.com/hbr731/hbr731.github.io/setup/pages/vi/profiles/hbr731.md

### CODE REVIEW

File: pages/vi/profiles/hbr731.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* Brooklyn, NY 
+* &#x1F556; EST
+* &#x1F4BB; Windows 10.
+
+### About Me
+Hi, I am an undergrad at Brooklyn College majoring in Computer Science and am graduating this Fall. I have used a raspberrypi
+to create a NAS server and a DNS server. Outside of class I like to learn new programming languages/frameworks and play badminton.
+
+### Dependencies
+<code> Coffee and my computer xD </code>
```
Comment (dogi) [inline pages/vi/profiles/hbr731.md:12]: please don't use html
markdown has its own syntax

File: pages/vi/profiles/hbr731.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Languages
+<code> Java, C#, C++, Assembly(MASM), Javascript, Python, Scheme </code>
```
Comment (dogi) [inline pages/vi/profiles/hbr731.md:15]: same as above

File: pages/vi/profiles/hbr731.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+### Framefworks/Libraries
+<code> React, React-Redux, Spring, Node.js, Express </code>
```
Comment (dogi) [inline pages/vi/profiles/hbr731.md:18]: same as above

File: pages/vi/profiles/hbr731.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+### Databases
+<code> MongoDB, SQL, PostgreSQL </code>
```
Comment (dogi) [inline pages/vi/profiles/hbr731.md:21]: same as above

**Review by dogi — CHANGES_REQUESTED** [review]

**Comment by dogi** [comment]
also it would be nice you add a link or a linked picture

**Review by lmmrssa — COMMENTED** [review]
@hbr731 some more markdown syntax will look nice

**Comment by vmnet8** [comment]
Also please put the raw.githack.com link.

**Pull request closed without merging**